src/Pages/Playlist/Playlist.jsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { Navbar } from "../../Components/Navbar/Navbar";
+import { Sidebar } from "../../Components/Sidebar/Sidebar";
+import { useAuth } from "../../Context/Auth/auth-context";
+import { usePlaylist } from "../../Context/PlaylistContext/PlaylistContext";
+import { BsThreeDotsVertical, CgPlayListCheck } from "../../Utils/icons";
+import { deletePlaylist } from "../../Services/playlistService";
+
+const Playlist = () => {
+  const {
+    playlistState: { playlists },
+    dispatchPlaylist,
+  } = usePlaylist();
+  const navigate = useNavigate();
+  const [toggleOption, setToggle] = useState(false);
+  const { auth } = useAuth();
+  const deletePlaylistFunc = async (id) => {
+    try {
+      const data = await deletePlaylist(id, auth.tokenValue);
+      dispatchPlaylist({ type: "DELETE_PLAYLIST", payload: data.playlists });
+    } catch (error) {
+      console.log("error in deleting the whole playlist function", error);
+    }
+  };
+  return (
+    <div>
```

Comment:
suggestion:- use Fragment will not add any additional elements into the DOM tree, whereas, using a div will add a div to the DOM tree.